assets/content/cookbook/Other/search-for-files-with-extension.md

```diff
@@ -0,0 +1,42 @@
+[tags]: / "filesystem, sys, file"
```

Comment:
I am guessing there should be no spaces, see e.g. https://raw.githubusercontent.com/HaxeFoundation/code-cookbook/master/assets/content/cookbook/Macros/enum-abstract-values.md